src/ruby/yast/path.rb

```diff
@@ -119,7 +119,7 @@ def load_components(value)
 
     def invalid_buffer?(buffer)
       if buffer.start_with?("-") || buffer.end_with?("-")
-        Yast.y2error "Cannot have dash before or after dot '#{value}'"
+        Yast.y2error "Cannot have dash before or after dot '#{buffer}'"
```

Comment:
you mean "can be" discovered? and to be honest you cannot be ever sure, as method `value` can be dynamically defined ;)